cpp/include/cugraph/utilities/thrust_tuple_utils.hpp

```diff
@@ -169,6 +171,14 @@ struct is_std_tuple_of_arithmetic_vectors<std::tuple<rmm::device_uvector<Ts>...>
   static constexpr bool value = (... && std::is_arithmetic_v<Ts>);
 };
 
+template <typename T>
+struct is_std_tuple_of_arithmetic_device_spans : std::false_type {};
+
+template <typename... Ts>
+struct is_std_tuple_of_arithmetic_device_spans<std::tuple<raft::device_span<Ts>...>> {
+  static constexpr bool value = (... && std::is_arithmetic_v<Ts>);
+};
```

Comment:
To be consistent, I think we should rename this function to "is_std_tuple_of_arithmetic_spans" or rename the above "is_std_tuple_of_vectors" to "is_std_tuple_of_device_vectors".

Reply:
I am a bit inclined to renaming this function as in some years, the distinction between the host vector and device vector may get significantly blurred.